client/structure/login.js

```diff
@@ -40,7 +40,13 @@ export function LoginScreen({ action }) {
         const personaPreview = await getFirebaseDataAsync([
             'silo', datastore.getSiloKey(), 'structure', 'profile',
             'instance', personaKey, 'global', 'preview']);
+        const fbUser = datastore.getFirebaseUser();            
         if (!personaPreview?.name) {
+            const isCelebrity = await datastore.callServerAsync(
```

Comment:
I think we probably actually need to do the celebrity check as part of getOrCreateUser whenever a user is created, as well as in whatever server side call updates the name.

A user who is banging the server API could potentially bypass the "set up your profile" screen, causing their name to default to whatever the SSO provider set.

Reply:
as discussed, will do this later.